cmake/projects/magnum/hunter.cmake

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+
+
+
+hunter_add_version(
+    PACKAGE_NAME
+    magnum
+    VERSION
+    2019.01
+    URL
+    "https://github.com/mosra/magnum/archive/v2019.01.tar.gz"
+    SHA1
+    af356adc9e83826d55bd7f438c29f98104f95fd0
+)
+
+
+# CMAKE_SYSTEM_NAME dependant default options below
+
+# Take all the plugins
+set(MAGNUM_CMAKE_ARGS
```

Comment:
This variable will leak to the user space, it's better to use `_hunter_*` instead, e.g.:
- https://github.com/ruslo/hunter/blob/v0.23.99/cmake/projects/libjson-rpc-cpp/hunter.cmake#L35-L45